hydra/garaga/starknet/groth16_contract_generator/parsing_utils.py

```diff
@@ -18,9 +18,9 @@
 from garaga.precompiled_circuits.multi_miller_loop import MultiMillerLoopCircuit
 
 # https://github.com/risc0/risc0-ethereum/blob/main/contracts/src/groth16/ControlID.sol
-RISC0_CONTROL_ROOT = 0x8B6DCF11D463AC455361B41FB3ED053FEBB817491BDEA00FDB340E45013B852E
+RISC0_CONTROL_ROOT = 0xA516A057C9FBF5629106300934D48E0E775D4230E41E503347CAD96FCBDE7E2E
 RISC0_BN254_CONTROL_ID = (
-    0x05A022E1DB38457FB510BC347B30EB8F8CF3EDA95587653D0EAC19E1F10D164E
+    0x0EB6FEBCF06C5DF079111BE116F79BD8C7E85DC9448776EF9A59AAF2624AB551
 )
 
```

Comment:
This shouldn't change. What happened ?

Reply:
I was debugging my program and didn't see that this it was included in the commit.

Reply:
Also, I am not sure they are the latest values. The latest values should be those [here](https://docs.rs/risc0-circuit-recursion/1.1.2/src/risc0_circuit_recursion/control_id.rs.html#15-180). You can check [Issue#234](https://github.com/keep-starknet-strange/garaga/issues/234) I have explained my train of thoughts there.